### ISSUE TITLE
Unable to tcprewrite range of ports with --portmap

### ISSUE THREAD

**Issue by sdubrul** [appneta/tcpreplay#422]
The following doesn't seem to work in 4.2.5 on centos 7. In previous versions the portmap would allow for a range of original ports to rewrite a given port.

```
/bin/tcprewrite -r 1-100:49148 --infile=in.cap --outfile=out.cap --fixcsum
Fatal Error: Unable to parse args: From parse_args.c:tcpedit_post_args() line 181:
Unable to parse --portmap=1-100:4900
```

**Comment by fklassen** [comment]
Thanks for the bug report. Scheduled for 4.2.7

**Comment by fklassen** [comment]
Fixed in PR #443
